src/components/RegisterModal/RegisterModal.js

```diff
@@ -84,9 +85,22 @@ class RegisterModal extends Component {
       alert("Please verify that you are a human !!!");
   };
 
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFields((err, values) => {
+      if (!err) {
+        this.handleCreate();
+      }
+    });
+  };
+
   onChange = e => {
     this.setState({ [e.target.name]: e.target.value });
   };
+  onChangeSwitch = (checked,e) => {
+    let state={...this.state, isVendor: checked}
+    this.setState(state);
```

Comment:
@ashwaniYDV `let state={...this.state, isVendor: checked}` is not required 
Just write `this.setState({isVendor:checked})`

Reply:
I've done it inthat way first but it was showing errors

Reply:
because swich change event is on another function

Reply:
What error ?

Reply:
sorry its ok now.
I'll update

Reply:
done